src/h5web/toolbar/heatmap/ColorMapSelector.tsx

```diff
@@ -4,9 +4,15 @@ import { Wrapper, Button, Menu, MenuItem } from 'react-aria-menubutton';
 import { MdArrowDropDown } from 'react-icons/md';
 import { useWindowSize } from 'react-use';
 import styles from './ColorMapSelector.module.css';
-import { INTERPOLATOR_GROUPS, INTERPOLATORS } from './interpolators';
-import type { ColorMap, D3Interpolator } from './models';
-import { generateCSSLinearGradient } from './utils';
+import {
+  INTERPOLATOR_GROUPS,
+  INTERPOLATORS,
+} from '../../visualizations/heatmap/interpolators';
+import type {
+  ColorMap,
+  D3Interpolator,
+} from '../../visualizations/heatmap/models';
+import { generateCSSLinearGradient } from '../../visualizations/heatmap/utils';
```

Comment:
This is the main downside: importing utilities and models from toolbar components is a bit more tedious.